src/amo/components/AddonSummaryCard/index.js

```diff
@@ -0,0 +1,87 @@
+/* @flow */
+import * as React from 'react';
+import { compose } from 'redux';
+
+import AddonTitle from 'amo/components/AddonTitle';
+import Link from 'amo/components/Link';
+import RatingsByStar from 'amo/components/RatingsByStar';
+import translate from 'core/i18n/translate';
+import { getAddonIconUrl } from 'core/imageUtils';
+import Card from 'ui/components/Card';
+import LoadingText from 'ui/components/LoadingText';
+import Rating from 'ui/components/Rating';
+import type { AddonType } from 'core/types/addons';
+import type { I18nType } from 'core/types/i18n';
+
+import './styles.scss';
+
+type Props = {|
+  addon: AddonType | null,
+  headerText: string,
+|};
+
+type InternalProps = {|
+  ...Props,
+  i18n: I18nType,
+|};
+
+export class AddonSummaryCardBase extends React.Component<InternalProps> {
```

Comment:
I don't see any lifecycle methods so you could make this a stateless component. React offers some optimizations internally for those so it's always a good idea.